src/screens/ProfileScreen/elements/RenderItem.js

```diff
@@ -274,15 +292,14 @@ const styles = StyleSheet.create({
     width: '100%',
     height: dimen.size.PROFILE_ITEM_HEIGHT,
     maxHeight: dimen.size.PROFILE_ITEM_HEIGHT,
-    shadowColor: COLORS.lightgrey,
-    shadowOffset: {
-      width: 1,
-      height: 8
-    },
-    shadowOpacity: 0.5,
     backgroundColor: COLORS.white,
     paddingBottom: 0,
-    borderBottomColor: 'transparent'
+    borderBottomColor: 'transparent',
+    marginBottom: normalizeFontSizeByWidth(4),
+    shadowColor: COLORS.black000,
+    shadowOffset: {width: 0, height: 4},
+    shadowOpacity: 0.3,
+    shadowRadius: 4
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made to the styles here seem to be primarily aesthetic (changing colors, shadows, etc.). As long as these changes align with the desired look and feel of the application, they should be fine.



<!-- This is an auto-generated comment by OSS CodeRabbit -->